### ISSUE TITLE
check_sy_subrc, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1352]
```abap
      SELECT SINGLE object FROM tadir INTO lv_object
        WHERE pgmid = 'R3TR'
        AND object IN ('CLAS','ENHS','CUS0','CUS1','CUS2')
        AND obj_name = iv_encl_object.
    ENDIF.
    IF sy-subrc <> 0.
```

**Comment by larshp** [comment]
```abap
    READ TABLE lt_results WITH KEY object = 'DTEL' obj_name = 'XMILOGID' TRANSPORTING NO FIELDS.
    cl_abap_unit_assert=>assert_subrc( ).
```
